Prospect-iris/src/test/java/com/temenos/marketplace/TestTeller_LcyCashinCashwdlITCase.java

```diff
@@ -0,0 +1,137 @@
+package com.temenos.marketplace;
+
+import static org.junit.Assert.*;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+import org.junit.Test;
+
+import com.temenos.useragent.generic.DefaultInteractionSession;
+import com.temenos.useragent.generic.InteractionSession;
+import com.temenos.useragent.generic.mediatype.AtomPayloadHandler;
+
+/**
+ * TODO: Document me!
+ *
+ * @author jfiricel
+ *
+ */
+public class TestTeller_LcyCashinCashwdlITCase {
+
+    //the user below is a teller one that should be present into the testing area, otherwise it should be changed with another one
+    public static String TELLER_USER_NAME = "CSAGENT";
```

Comment:
Hi @jfiricel 
Can you please remove this TELLER_USER_NAME and get it from Configuration file